src/components/views/messages/TextualBody.tsx

```diff
@@ -375,6 +380,7 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
         if (this.state.links.length && !this.state.widgetHidden && this.props.showUrlPreview) {
             widgets = (
                 <LinkPreviewGroup
+                    id={this.props.id}
```

Comment:
This is invalid and will cause two elements to be rendered with the same ID. IDs in DOM must be unique. https://developer.mozilla.org/en-US/docs/Web/API/Element/id

`widgets` is rendered alongside `body` and both are sharing the id.